src/main/java/racingcar/domain/Car.java

```diff
@@ -5,62 +5,46 @@
 
 public class Car implements Comparable<Car> {
 
-    private static final int RUNNABLE_BOUND = 4;
-    private static final int MAX_VALUE = 9;
-    private static final int MIN_VALUE = 0;
-
-    private CarName name;
-    private int distance;
-    private Randomable randomable;
+    private final CarName name;
+    private final Position position;
 
     public Car(String name) {
         this.name = new CarName(name);
-        this.distance = 0;
-        this.randomable = new RandomNumber();
+        this.position = new Position(0);
     }
```

Comment:
다른 생성자를 호출하여 처리해볼 수 있을 것 같아요.
```suggestion
    public Car(String name) {
        this(name, 0);
    }
```